packages/contracts-bedrock/src/dispute/OutputBisectionGame.sol

```diff
@@ -173,7 +180,7 @@ contract OutputBisectionGame is IOutputBisectionGame, Clone, ISemver {
         // SAFETY:    While the `attack` path does not need an extra check for the post
         //            state's depth in relation to the parent, we don't need another
         //            branch because (n - n) % 2 == 0.
-        bool validStep = VM.step(_stateData, _proof, uuid) == Claim.unwrap(postState.claim);
+        bool validStep = VM.step(_stateData, _proof, Hash.unwrap(uuid)) == Claim.unwrap(postState.claim);
```

Comment:
Feels like the `Hash.unwrap` just creates unnecessary friction - should `VM.step` accept a `Hash`?

Reply:
Spoke offline - pushing this off due to the `MIPS` contract not respecting the dispute contracts' UDTs.